src/App.js

```diff
@@ -1,15 +1,24 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './App.css';
-import Header from './layout/Header.jsx';
-import MarketPage from './pages/market/Market.jsx';
+import Header from './Layout/Header';
+import Market from './pages/market/Market';
+import HomePage from './pages/home/HomePage.jsx';
+import AddItemPage from './pages/AddItem/AddItemPage.jsx';
 
 function App() {
   return (
-    <>
+    <BrowserRouter>
       <Header />
+
       <main>
-        <MarketPage />
+        <Routes>
+          <Route index element={<HomePage />} />
+          <Route path='items' element={<Market />} />
+          <Route path='additem' element={<AddItemPage />} />
+          <Route path='*' element={<HomePage />} />
```

Comment:
```suggestion
          <Route index element={<HomePage />} />
          <Route path='items' element={<Market />} />
          <Route path='additem' element={<AddItemPage />} />
          <Route path='*' element={<HomePage />} />
```

* index는 기본경로로 지정되는거라 `<Route path="/" element={<HomePage />} />` 이것과 같습니다.
* `*` 전체 선택자는 지정되지 않은 모든 경로를 뜻해서, 페이지를 찾을 수 없습니다 같은 에러페이지를 띄우는 용도로 많이 씁니다.